docs/src/pages/guides/troubleshoot.md

```diff
@@ -5,6 +5,14 @@ title: Troubleshoot
 
 If you have problem installing using `npm`, please check this [issue #216](https://github.com/hyperjumptech/monika/issues/216). It might be related to permission issue. In most cases, it can be solved by running
 
-```
+```bash
 sudo npm i -g @hyperjumptech/monika --unsafe-perm=true --allow-root
 ```
+
+If you have issues probing a particular target we suggest:
+
+1. Try sending the request through curl or a separate tool such as Insomnia.
+2. Isolate the problem probes in your `monika.yml` configuration file, perhaps try the probe alone removing others while you troubleshoot.
```

Comment:
Isolate the problematic probes in your `monika.yml` configuration file by removing the other probes while you troubleshoot.

Reply:
done